source/MissionPanel.cpp

```diff
@@ -491,7 +517,9 @@ bool MissionPanel::Hover(int x, int y)
 		if(static_cast<int>(index) < AcceptedVisible())
 			dragSide = 1;
 	}
-	return dragSide ? true : MapPanel::Hover(x, y);
+	hoverSort = y < Screen::Top() + 30 && y >= Screen::Top() + 10 &&
+			x < Screen::Left() + SIDE_WIDTH - 25 && x >= Screen::Left() + SIDE_WIDTH - 45;
+	return (dragSide || hoverSort) ? true : MapPanel::Hover(x, y);
```

Comment:
Is this change required? I am not sure, but I think that if it is selected we will fall into the first if and return false/true already

Reply:
Yea that's a little redundant. 

But the chunk setting hoverSort is still used elsewhere.